index.html

```diff
@@ -267,12 +266,12 @@ <h2 class="primary-text dark center-text margin-zero">About Me</h2>
       <div class="form">
         <h2 class="primary-text dark center-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         <form class="form-container" action="https://formspree.io/f/mzbqodyb" method="POST">
-          <input class="dekstop-hidden" placeholder="Full name" type="text" name="full_name" maxlenght="30" required />
+          <input class="dekstop-hidden" placeholder="Full name" type="text" name="full_name" maxlength="30" required />
           <div class="first-last-name-container mobile-hidden">
-            <input placeholder="First name" type="text" name="first_name" maxlenght="30" required />
-            <input placeholder="Last name" type="text" name="last_name" maxlenght="30" required />
+            <input placeholder="First name" type="text" name="first_name" maxlength="30" required />
+            <input placeholder="Last name" type="text" name="last_name" maxlength="30" required />
           </div>
-          <input type="email" name="email" placeholder="Email address" required />
+          <input type="email" name="email" placeholder="Email address" pattern="[^A-Z\s]+$" required />
           <textarea name="message" maxlenght="500" placeholder="Hey I’d like to connect to..." required></textarea>
           <button class="button card-button white width-fit self-center" type="submit">Get in touch</button>
```

Comment:
- @ichsansandy The requirements state that:

-  `error message should show up close to the submit button or the input fields. (i.e error message can show up around the input fields or just around the submit button)`.
-   `the submit button/input `"click"` event is used to attach the function checking the email format.`

Please make the error message show up close to the button as needed in the requirement and when the submit button is clicked it is used to attach a function that checks if the email is in lowercase letter.